### PR TITLE
Admin capabilities

### PR DESCRIPTION
**Pull request by weronica**
- Users can no longer make an account on the site unless they have been invited
- Users can update their first name and last name
- Users can update their email address
- Removed unnecessary logic and templates

(Out of the box) Admins can invite other admins to join the site. Confirmation email will be sent, and invited admin can set their new password.

r @armant

### CODE REVIEW

File: app/account/forms.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
-from flask import url_for
 from flask.ext.wtf import Form
 from wtforms.fields import (
-    StringField,
     PasswordField,
```
Comment (armant) [inline app/account/forms.py]: Do we need this? Doesn't seem to be used in this file.

File: app/account/forms.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
     BooleanField,
```
Comment (armant) [inline app/account/forms.py]: Do we need this? Doesn't seem to be used in this file.

Comment (weronica) [inline app/account/forms.py]: You can't see the entire file in this diff. `BooleanField` is used in `LoginForm`. 

As a code reviewer, you don't need to worry about unnecessary import statements. `flake8` checks to make sure that all imports are actually being used. :)

File: app/account/forms.py (hunk 1 above)
Comment (weronica) [inline app/account/forms.py]: See my comment below.

**Comment by armant** [comment]
r @dzhang55

File: app/account/forms.py (hunk 2 above)
Comment (armant) [inline app/account/forms.py]: Oh I see, thanks!

File: app/account/forms.py (hunk 1 above)
Comment (armant) [inline app/account/forms.py]: Seen!

File: app/account/views.py [hunk 3]
```diff
@@ -11,12 +11,12 @@
...
 from .forms import (
     LoginForm,
-    RegistrationForm,
     CreatePasswordForm,
     ChangePasswordForm,
     ChangeEmailForm,
     RequestResetPasswordForm,
-    ResetPasswordForm
+    ResetPasswordForm,
+    ChangeAccountInfoForm
```
Comment (armant) [inline app/account/views.py]: Let's have imports in alphabetical order.

File: app/account/forms.py (hunk 2 above)
Comment (armant) [inline app/account/forms.py]: Let's have imports in alphabetical order.

File: manage.py [hunk 4]
```diff
@@ -64,6 +64,13 @@ def setup_dev():
     """Runs the set-up needed for local development."""
     setup_general()
 
+    admin_email = 'm4a@gmail.com'
```
Comment (armant) [inline manage.py]: Is there a way to do this without hardcoding? Maybe import from config file?

**Comment by armant** [comment]
@weronica please see comments.

File: app/account/forms.py (hunk 2 above)
Comment (weronica) [inline app/account/forms.py]: Done

File: app/account/views.py (hunk 3 above)
Comment (weronica) [inline app/account/views.py]: Done

File: app/account/views.py [hunk 5]
```diff
@@ -132,6 +112,25 @@ def change_password():
...
+    """
+    form = ChangeAccountInfoForm()
+    if form.validate_on_submit():
+        current_user.first_name = form.first_name.data
+        current_user.last_name = form.last_name.data
+        db.session.add(current_user)
+        db.session.commit()
+    else:
+        form.first_name.data = current_user.first_name
+        form.last_name.data = current_user.last_name
```
Comment (dzhang55) [inline app/account/views.py]: What does this else case do?

File: manage.py (hunk 4 above)
Comment (weronica) [inline manage.py]: Done.

File: app/account/views.py (hunk 5 above)
Comment (weronica) [inline app/account/views.py]: Added a comment - "When form is first displayed, pre-populate its fields with the user's current information."

**Comment by weronica** [comment]
@armant @dzhang55 I addressed all of your comments

**Pull request merged**